packages/slice-machine/server/src/api/state.ts

```diff
@@ -104,7 +76,11 @@ export default async function handler(
   };
 
   return {
-    ...serverState,
+    customTypes: serverState.customTypes,
+    remoteCustomTypes: serverState.remoteCustomTypes,
+    libraries: serverState.libraries,
+    remoteSlices: serverState.remoteSlices,
+    clientError: serverState.clientError,
```

Comment:
do we still need to send this property ?
It seems only used in the backend on the push handler.

Reply:
I think we can can also remove it